thirdeye/thirdeye-frontend/app/styles/pods/custom/parent-anomalies-table.scss

```diff
@@ -0,0 +1,20 @@
+.parent-anomalies-table {
+    width: 100%;
+    border: 1px solid #ccc;
+    tr {
+        border-bottom: 1px solid #ccc;
+    }
+    td, th {
+        padding: 10px 10px 10px 20px;
+
+        p {
```

Comment:
To prevent issues in the later, please use a class instead of p element selector directly, otherwise, any p tag will have this style.

Reply:
I thought of that too - thanks for the reminder